README.md

```diff
@@ -54,6 +54,7 @@ Currently it shows:
 * Current Python pyenv (`🐍`).
 * Current .NET SDK version, through dotnet-cli (`.NET`).
 * Current Ember.js version, through ember-cli (`🐹`).
+* Current AWS-cli profile (`☁️`) ([Using named profiles](http://docs.aws.amazon.com/cli/latest/userguide/cli-multiple-profiles.html))
```

Comment:
Try to follow the same order everywhere. Since `aws` section is listed after docker in example, Better to follow that order everywhere.